Direct_Commands/BlueSparkle.json

```diff
@@ -1,19 +1,11 @@
 {
     "animation": {
+        "typename": "sequence",
         "animations": [
-            "Fade.Fade"
```

Comment:
I deleted all of this just to make the "minimal working example", and because the settings you had didn't do anything (were the same as the default).